### PR TITLE
Prs examples

### PR DESCRIPTION
**Pull request by alainli0928**
# Description
Add new PRS partition key example and move all PRS examples to new location. 

# Checklist

- [ ] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: cli/jobs/parallel/1a_oj_sales_prediction/src/parallel_train/parallel_train.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    )
+    print("##############################################")
+    print("relative_path:")
+    print(relative_path)
+    print("##############################################")
+
+    if not os.path.exists(relative_path):
+        os.makedirs(relative_path)
+
+    with open(os.path.join(relative_path, model_file_name), "wb") as f:
```
Comment (cloga) [inline cli/jobs/parallel/1a_oj_sales_prediction/src/parallel_train/parallel_train.py:111]: It is better to leverage mlflow to do model metrics logging and log model: 
- https://learn.microsoft.com/en-us/azure/machine-learning/how-to-log-mlflow-models?tabs=wrapper
- https://learn.microsoft.com/en-us/azure/machine-learning/how-to-log-view-metrics?tabs=interactive

File: cli/jobs/parallel/1a_oj_sales_prediction/src/partition_data/partition_data.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
```
Comment (cloga) [inline cli/jobs/parallel/1a_oj_sales_prediction/src/partition_data/partition_data.yml:1]: Seems this worth to be a built-in component in public registry.

File: sdk/python/jobs/parallel/1a_oj_sales_prediction/src/parallel_train/parallel_train.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    )
+    print("##############################################")
+    print("relative_path:")
+    print(relative_path)
+    print("##############################################")
+
+    if not os.path.exists(relative_path):
+        os.makedirs(relative_path)
+
+    with open(os.path.join(relative_path, model_file_name), "wb") as f:
```
Comment (cloga) [inline sdk/python/jobs/parallel/1a_oj_sales_prediction/src/parallel_train/parallel_train.py:111]: Better change to leverage mlflow to log model

**Review by cloga — APPROVED** [review]

File: cli/jobs/parallel/1a_oj_sales_prediction/src/partition_data/partition_data.yml (hunk 2 above)
Comment (alainli0928) [reply cli/jobs/parallel/1a_oj_sales_prediction/src/partition_data/partition_data.yml:1]: Yeah, maybe we could consider registry later. Now it has potential risk on bugs :P

**Review by bupt-wenxiaole — APPROVED** [review]

File: sdk/python/jobs/parallel/1a_oj_sales_prediction/oj_sales_prediction.ipynb [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+    "    )\n",
+    "\n",
+    "\n",
+    "# Create pipeline instance\n",
+    "my_job = partition_job_in_pipeline(\n",
+    "    pipeline_input_data=input_oj_data,\n",
+    ")\n",
+    "\n",
+    "# Set pipeline level compute\n",
+    "my_job.tags.update\n",
```
Comment (wangchao1230) [inline sdk/python/jobs/parallel/1a_oj_sales_prediction/oj_sales_prediction.ipynb:289]: why need this line: 
"my_job.tags.update"

File: sdk/python/jobs/parallel/1a_oj_sales_prediction/oj_sales_prediction.ipynb [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+    "        date_col=\"WeekStarting\",\n",
+    "        lagging_orders=\"1,2,3,4,5,6\",\n",
+    "    )\n",
+    "\n",
+    "    # User could override parallel job run-level property when invoke that parallel job/component in pipeline.\n",
+    "    parallel_train.resources.instance_count = 3\n",
+    "    parallel_train.max_concurrency_per_instance = 2\n",
+    "    parallel_train.mini_batch_error_threshold = 10\n",
+    "    parallel_train.outputs.model_perf.path = (\n",
+    "        f\"azureml://datastores/workspaceblobstore/paths/${{name}}/my_append_output.csv\"\n",
```
Comment (wangchao1230) [inline sdk/python/jobs/parallel/1a_oj_sales_prediction/oj_sales_prediction.ipynb:279]: use ${{default_datastore}}

File: sdk/python/jobs/parallel/3a_mnist_batch_identification/mnist_batch_prediction.ipynb [hunk 6]
```diff
@@ -0,0 +1,379 @@
...
+     "logged": 1675653943687
+    },
+    "name": "workspace"
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Found the config file in: /config.json\n"
```
Comment (wangchao1230) [inline sdk/python/jobs/parallel/3a_mnist_batch_identification/mnist_batch_prediction.ipynb:100]: seems  addtional output here

File: sdk/python/jobs/parallel/3a_mnist_batch_identification/mnist_batch_prediction.ipynb [hunk 7]
```diff
@@ -0,0 +1,379 @@
...
+    "gather": {
+     "logged": 1675654944788
+    },
+    "name": "submit-pipeline"
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table style=\"width:100%\"><tr><th>Experiment</th><th>Name</th><th>Type</th><th>Status</th><th>Details Page</th></tr><tr><td>parallel examples</td><td>teal_star_wbf1qth7wy</td><td>pipeline</td><td>Preparing</td><td><a href=\"https://ml.azure.com/runs/teal_star_wbf1qth7wy?wsid=/subscriptions/ee85ed72-2b26-48f6-a0e8-cb5bcf98fbd9/resourcegroups/alainli-rg/workspaces/alainli-prs-eastus2&amp;tid=72f988bf-86f1-41af-91ab-2d7cd011db47\" target=\"_blank\" rel=\"noopener\">Link to Azure Machine Learning studio</a></td></tr></table>"
```
Comment (wangchao1230) [inline sdk/python/jobs/parallel/3a_mnist_batch_identification/mnist_batch_prediction.ipynb:305]: remove output

File: sdk/python/jobs/parallel/1a_oj_sales_prediction/oj_sales_prediction.ipynb (hunk 4 above)
Comment (alainli0928) [reply sdk/python/jobs/parallel/1a_oj_sales_prediction/oj_sales_prediction.ipynb:289]: Not sure where I copied this. Seems not useful. Will remove this from next commit.

File: sdk/python/jobs/parallel/1a_oj_sales_prediction/oj_sales_prediction.ipynb (hunk 5 above)
Comment (alainli0928) [reply sdk/python/jobs/parallel/1a_oj_sales_prediction/oj_sales_prediction.ipynb:279]: Should be 'azureml://datastores/${{default_datastore}}/paths/...' ?

**Pull request merged**